sandy/sections/mf2.py

```diff
@@ -0,0 +1,860 @@
+                 {'ER': 171.4, 'AJ': 0.5, 'GT': 0.0509, 'GN': 0.0253, 'GG': 0.0256, 'GF': 0.0}, 
+                 {'ER': 184.2, 'AJ': 0.5, 'GT': 0.0489, 'GN': 0.0233, 'GG': 0.0256, 'GF': 0.0}, 
+                 {'ER': 195.0, 'AJ': 0.5, 'GT': 0.0709, 'GN': 0.0453, 'GG': 0.0256, 'GF': 0.0}, 
+                 {'ER': 209.0, 'AJ': 0.5, 'GT': 0.1009, 'GN': 0.0753, 'GG': 0.0256, 'GF': 0.0}, 
+                 {'ER': 226.0, 'AJ': 0.5, 'GT': 0.0484, 'GN': 0.0228, 'GG': 0.0256, 'GF': 0.0}, 
+                 {'ER': 241.0, 'AJ': 0.5, 'GT': 0.0311, 'GN': 0.0055, 'GG': 0.0256, 'GF': 0.0}, 
+                 {'ER': 248.0, 'AJ': 0.5, 'GT': 0.0809, 'GN': 0.0553, 'GG': 0.0256, 'GF': 0.0}]}     
+        
+    
+    Resonance parameters of the iron 58
+    LRU = 1 LRF = 3 
+    >>> tape = sandy.get_endf6_file("endfb_80", "xs", 260580)
+    >>> dic = sandy.read_mf2(tape, 2637)
+    >>> print (dic["NIS"]["ISO"][26058]["NER"][(1e-05, 350000.0)]["L"][0]["RES_PAR"][20])
+    {'ER': -3612.8, 'AJ': 0.5, 'GN': 271.55, 'GG': 0.74169, 'GFA': 0.0, 'GFB': 0.0}
+
+     
+    
+    Resonance parameters of the iron 58
+    LRU = 2 LFW = 0 LRF = 1
+    >>> tape = sandy.get_endf6_file("endfb_80", "xs", 260580)
+    >>> dic = sandy.read_mf2(tape, 2637)
+    >>> print (dic["NIS"]["ISO"][26058]["NER"][(350000.0, 3000000.0)]["L"][3])
+    {'AWRI': 57.43561, 'L': 3, 'NJS': 2, 
+     'RES_PAR': [{'D': 8033.33, 'AJ': 2.5, 'AMUN': 1.0, 'GNO': 0.482, 'GG': 0.7, '0': 0.0}, 
+                 {'D': 6025.0, 'AJ': 3.5, 'AMUN': 1.0, 'GNO': 0.3615, 'GG': 0.7, '0': 0.0}]}
+    
+    
+    Resonance parameters of the iron 54
+    LRU = 1 LRF = 7
+    >>> tape = sandy.get_endf6_file("endfb_80", "xs", 260540)
+    >>> dic = sandy.read_mf2(tape, 2625)
+    >>> print (dic["NIS"]["ISO"][26054]["NER"][(1e-05, 1036000.0)]["J"][2]["SPIN_GROUP"])
+    [{'PPI': 1.0, 'L': 0.0, 'SCH': 0.0, 'BND': 0.0, 'APE': 0.0, 'APT': 0.0}, 
+     {'PPI': 2.0, 'L': 1.0, 'SCH': 0.5, 'BND': 0.0, 'APE': 0.54373, 'APT': 0.54373}]
+    
+    
+    Resonance parameters of the Uranium 235
+    LRU = 2 LRF = 2
+    >>> tape = sandy.get_endf6_file("jeff_33", "xs", 922350)
+    >>> dic = sandy.read_mf2(tape, 9228)
+    >>> print (dic["NIS"]["ISO"][92235]["NER"][(2250.0, 46200.0)]['L'][0]["J"][0]["RES_PAR"][0])
+    {'ES': 2250.0, 'D': 1.058, 'GX': 0.0, 'GNO': 0.000107789, 'GG': 0.038513, 'GF': 0.40102}
+    
+    """
+    df = tape._get_section_df(mat, mf, mt)
+    out = {
+            "MAT": mat,         
+            "MF": mf,
+            "MT": mt,
+            }
+    i = 0
+    I = {}
+    C, i = sandy.read_cont(df, i)
+    add = {
+            "ZA": C.C1,             #designation for an isotope
+            "AWR": C.C2,            #AWR is defines as the ratio of the mass of the material to that of the neutron
+            "NIS": C.N1,            #NIS is the number of isotopes in the material
+            }
+    NIS = int(C.N1)
+    I.update(add)
+    NER = int(C.N1)
+    LFW = int(C.L2)
+    out.update({"INTRO": I})
```

Comment:
Why do we need ZA, AWR and NIS in a special section (INTRO).
Couldn't they just be in out, as 
```
out = {
   ZA:,
   AWR:,
   NIS:,
}
```